MyString.java

```diff
@@ -20,8 +20,16 @@ public static void main(String args[]) {
      * @return the number of times c appears in str
      */
     public static int countChar(String str, char ch) {
-        //// Replace the following statement with your code
-        return 0;
+        if(str == ""){

```

Comment:
1. Use `str.equals("")` or `str.length() == 0`
2. Not needed (code will result in same either way)